src/app/_sections/ExperiencesSection/Experience.tsx

```
@@ -0,0 +1,88 @@
+        ) : (
+          <></>
+        )}
+      </div>
+      <div>
+        <h5 className="mb-2 text-xl font-bold">주요 활동</h5>
+        <ul>
+          {activities.map((activity, idx) => (
+            <li
+              key={`act-${title}-${idx}`}
+              className="text-lg [&>b]:font-semibold"
+              dangerouslySetInnerHTML={{ __html: activity }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)